### PR TITLE
fix(plugin): name for field components

### PR DESCRIPTION
**Pull request by nourbalaha**
Done:
- Add more info to field components' name
- Display full name with long names

### CODE REVIEW

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx [hunk 1]
```diff
@@ -174,6 +207,8 @@ const BodyWrapper = styled(AccordionItemPanel)`
 const Title = styled.p`
   margin: 0;
   user-select: none;
+  width: 200px;
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx:210]: Is this 200px going to work in mobile as well?

Comment (nourbalaha) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx:207]: I think 200px is good for mobile as well. setting the width is necessary to make the paragraph take multiple lines

Comment (KaWaite) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx:207]: What if they have a smaller phone like an iphone 7 or iphone se? Is 200px going to be wider than their screen?

Comment (nourbalaha) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx:207]: iPhone 7's width is 375px and iPhone SE's width is 320px right?

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx [hunk 2]
```diff
@@ -96,7 +122,11 @@ const FieldComponent: React.FC<Props> = ({
                     {Field && (
                       <ArrowIcon icon="arrowDown" size={16} direction="right" expanded={expanded} />
                     )}
-                    <Title>{field.type === "template" ? field.name : fieldName[field.type]}</Title>
+                    <Title>
+                      {field.type === "template"
+                        ? `${field.name}(${getFieldGroup(field.type)})`
+                        : `${fieldName[field.type]}(${getFieldGroup(field.type)})`}
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx:128]: Can you build the name in a useMemo so it doesn't have to be calculated each render

**Review by KaWaite — APPROVED** [review]

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx (hunk 2 above)
Comment (nourbalaha) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/index.tsx]: Done!

**Pull request merged**